src/mixins/date-mixin.ts

```diff
@@ -167,4 +167,45 @@ export default class DateMixin extends Vue {
 
     return `${dateStr} at ${timeStr} Pacific time`
   }
+
+  /**
+   * Add a number of months to a date and return "YYYY-MM-DD".
+   * Date must be in the format of "YYYY-MM-DD" and months is a number
+   * @example (3, 2023-02-03) -> "2023-05-03"
+   * @example (18, 2023-02-03) -> "2024-08-03"
+   */
+  addMonthsToDate (month: number, date: string): string {
+    if (date === null || date === '') {
```

Comment:
Or you could do `if (!date)`.

Or you could do `date = date || '2023-01-01'`.

Or you could specify the default in the method signature:
`addMonthsToDate (month: number, date = '2023-01-01'): string {`

Reply:
Ditto for lines 194 and 197.